hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/config/converter/ProfileConverter.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.spi.config.converter;
+
+import com.hedera.node.app.spi.config.Profile;
+import com.swirlds.config.api.converter.ConfigConverter;
+
+/**
+ * Config api {@link ConfigConverter} implementation for the type {@link Profile}.
+ */
+public class ProfileConverter implements ConfigConverter<Profile> {
+
+    @Override
+    public Profile convert(final String value) throws IllegalArgumentException, NullPointerException {
+        if (value == null) {
+            throw new NullPointerException("null can not be converted");
+        }
+
+        try {
+            final int i = Integer.parseInt(value);
+            if (i == 0) {
```

Comment:
Switch statement?

We should get rid of the possibility to use numbers to define profiles. Do you know if there is already an issue for that?

Reply:
I just copied the original code from mono service. but yes, this one can be refactored. Since nobody else reviewed / approved this PR a code change would not be too bad.